bindings/nodejs/tests/client/examples.spec.ts

```diff
@@ -93,10 +93,6 @@ describe.skip('Main examples', () => {
         const addressOutputs = await client.getOutputs(outputIdsResponse.items);
 
         expect(addressOutputs).toBeDefined();
-
-        addressOutputs.forEach((output) => {
-            expect(output.metadata.blockId).toBeValidBlockId();
-        });
```

Comment:
why?

Reply:
Hmm ok, and we don't have a version with metadata in the bindings instead of removing tests?

Reply:
correct

Reply:
we could those methods as well oc